tests/unit/test_graph_parser.py

```diff
@@ -840,8 +841,33 @@ def test_fail_family_triggers_on_tasks(ftrig):
     gp = GraphParser()
     with pytest.raises(GraphParseError) as cm:
         gp.parse_graph(f"foo:{ftrig} => bar")
-        assert (
```

Comment:
[unrelated] Broken assertion.

Reply:
Am I blind or have you replaced it with an identical one?

Reply:
I've just dedented the assertion so that it gets run.

```python
with pytest.raises(Exception):
    raise Exception()
    print('this code will never get called')
print('this code will always get called')
```